### PR TITLE
Feat/add notes section to template

### PR DESCRIPTION
**Pull request by TeddyAndersson**
## Explain the changes you’ve made
Added a new notes section in the pdf to display informal messages from applicant

## Explain why these changes are made
These changes makes it possible for the applicant to inform an administrator about things that are not nessecary to submit in an applicant. This information needs to be in the pdf so that an administrator can be notified and correctly asses the application.

## Explain your solution
The soultion was create a new attribute in the template-data called notes. The notes attribute is an array of objects that has the follwing shape.
```JS
{
title: 'Some title to display',
text: 'The message submitted by the applicant'
}
```
In order to populate this object with information from the form answers submitted i filtered out the answer based on it's id `otherMessage` and then inserted the value as the value of the text attribute in the note object.

The final result for the notes attribute in the template-data should look like this for know.

```JS
{
  ...,
  notes: [
    {
      title: 'Some title to display',
      text: 'The message submitted by the applicant'
    }
  ],
  ...
}
```

This data is further used in the ekb-recurring.hbs template to render the content when we generate html in the html-pdf-ms service. So the template has been updated to support this. This is how the section for notes looks.

```HTML
...
<section>
        <table>
            {{#each notes}}
                <tr>
                    <td>{{this.title}}</td>
                    <td>{{this.text}}</td>
                </tr>
            {{/each}}
        </table>
    </section>
...
```

## How to test the changes?

1. Checkout this branch
2. Deploy the resources and services that are needed in order for the pdf flow to work.
3. Upload the new template `ekb-recurring.hbs` to the templates folder in your pdf-storage bucket 
4. Submit form answers that includes a message from the applicant from the app or via the api, make sure it's decrypted and has the status set to active:submitted.

## Was this feature tested in the following environments?
- [X] Personal AWS Sandbox

### CODE REVIEW

**Comment by sebastianthulin** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/helsingborg-stad/repositories/602c13c73b93a7537d767e421a43afaa/pull-request/245)

File: services/viva-ms/helpers/createRecurringCaseTemplateData.js [hunk 1]
```diff
@@ -59,6 +59,20 @@ export function createPersonsObject(persons, answers) {
...
+    const [noteAnswer] = filteredAnswers;
+    const note = {
+      title: 'Meddelande från sökande',
+      text: noteAnswer.value,
+    };
+    notes.push(note);
+  }
+  return notes;
+}
+
```
Comment (dannil76) [inline services/viva-ms/helpers/createRecurringCaseTemplateData.js:75]: Not sure if notes array will be populated with more than one item with this implementation?
Have a hard time to graps this :)

**Comment by helsingborgsstad** [comment]
Task linked: [CU-1jg0q8b Lägg till sektion för fritext överst i pdf-mallen](https://app.clickup.com/t/1jg0q8b)

File: services/viva-ms/helpers/createRecurringCaseTemplateData.js (hunk 1 above)
Comment (TeddyAndersson) [reply services/viva-ms/helpers/createRecurringCaseTemplateData.js:75]: If you look at the pdf design the uppermost section of the pdf contains two "rows", But i'm uncertain what content goes into the first "row".

The two type of notes:
Har något ändrat sedan senast
Meddelande från sökande

Comment (TeddyAndersson) [reply services/viva-ms/helpers/createRecurringCaseTemplateData.js:75]: So currently the array only consits of one object, but the rendering supports multiple objects for the "notes" section.

**Review by Sven65 — APPROVED** [review]

**Review by dannil76 — APPROVED** [review]
Looks good

**Pull request merged**